Marlin/ultralcd.cpp

```diff
@@ -830,14 +848,13 @@ static void _lcd_move(const char* name, AxisEnum axis, int min, int max) {
     if (min_software_endstops) NOLESS(current_position[axis], min);
     if (max_software_endstops) NOMORE(current_position[axis], max);
     encoderPosition = 0;
-    if (movesplanned() <= 3)
-      line_to_current(axis);
+    manual_move_to_current(axis);
     lcdDrawUpdate = 1;
   }
   if (lcdDrawUpdate) lcd_implementation_drawedit(name, ftostr31(current_position[axis]));
   if (LCD_CLICKED) {
-      line_to_current(axis);
-      lcd_goto_menu(lcd_move_menu_axis);
+    manual_move_to_current(axis);
+    lcd_goto_menu(lcd_move_menu_axis);
   }
```

Comment:
It's probably very rare, but possible that a move that couldn't be added may only get added after a second or two delay when the menu exits. The main thing I wanted to get in place and try was the `manage_manual_move` function to see if there was some way to make manual moves interruptible. So if you changed your mind and spun in the other direction there would only be a small move before it reversed. But I have yet to work out the details.